testing/terraform/aws.tf

```diff
@@ -1,19 +1,142 @@
 variable "build_number" {}
 
+variable "amis" {
+  default = {
+    us-east-1      = "ami-6d1c2007"
+    us-west-2      = "ami-d2c924b2"
+    us-west-1      = "ami-af4333cf"
+    eu-central-1   = "ami-9bf712f4"
+    eu-west-1      = "ami-7abd0209"
+    ap-southeast-1 = "ami-f068a193"
+    ap-southeast-2 = "ami-fedafc9d"
+    ap-northeast-1 = "ami-eec1c380"
+    sa-east-1      = "ami-26b93b4a"
+  }
+}
+variable "availability_zones"  {
+  default = "a,b,c"
+}
+variable "control_count" { default = 3 }
+variable "datacenter" {default = "aws-us-west-2"}
+variable "edge_count" { default = 1 }
+variable "region" {default = "us-west-1"}
+variable "short_name" {default = "mantl-ci-${var.build_number}"}
+variable "long_name" {default = "mantl-ci-${var.build_number}"}
```

Comment:
Oh yeah, I forgot you can use variables in defaults. Thanks for the pointer! <-- still a newbie to terraform